### PR TITLE
added recipe list

### PR DESCRIPTION
**Pull request by shereemorrison**

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -1 +1,121 @@
 /* Styles sheet for the home page or any page with shared styles */
+body {
+    font-family:Cambria, Cochin, Georgia, Times, 'Times New Roman', serif
```
Comment (Fred-Khan) [inline index.css:3]: Missing a semicolon at the end.

**Review by Fred-Khan — CHANGES_REQUESTED** [review]
1. Assets such as images should go into an images folder so that they do not clutter up the root folder.

2. The JavaScript code needs to be refactored and ensure that it is modularised. Put all code blocks into functions that are associated with specific tasks to ensure code re-usability. This is so that the code becomes more readable, maintainable, and easier to understand. 

An example that would be easy to follow at a glance is as follows:
document.addEventListener('DOMContentLoaded', () => {
    populateAdviceList();
    setupSlider();
    setupCategoryNavigation();
});

The code shows that when the page is loaded it is calling the 3 separate function to do the heavy lifting. This would make the code easier to follow as the 3 functions do not need to be in the load function cluttering it up. The next developer can add their code and then call it in the load function.

3. CSS needs to be validated. Search for CSS Validator online and copy paste your CSS into it to detect any errors.

4. HTML Validation is recommended. Search for HTML Validator online and copy paste your CSS into it to detect any recommendations, warnings and/or errors.

5. WCAG consideration is also recommended.

**Comment by hebaDawahra** [comment]
I'm working on recipe list, and this is what i've done until now

**Comment by shereemorrison** [comment]
Incorrectly requested pull request - this was meant to be a clone instead

**Pull request closed without merging**